test/js/ui/map.test.js

```diff
@@ -292,6 +292,53 @@ test('Map', function(t) {
         }
     });
 
+    t.test('#setMinZoom', function(t) {
+        var map = createMap({zoom:0});
+        map.setMinZoom(3.5);
+        t.deepEqual(map.options.minZoom, 3.5);
+        t.end();
+    });
+
+    t.test('unset minZoom', function(t){
+        var map = createMap({minZoom:5});
```

Comment:
Let's pass a default `zoom: 5`  and assert that `t.deepEqual(map.getZoom(), 0);` to be 100% sure that this test is working the way that we expect